test/services/activation/services/eMailAddress/index.test.js

```diff
@@ -68,9 +50,11 @@ describe('activation/services/eMailAddress EMailAdresseActivationService', () =>
 			password,
 		};
 
-		const cb = getNotificationMock();
+		const nestMailService = { send: sinon.spy() };
+		app.services['nest-mail'] = nestMailService;
+
 		const res = await activationService.create(data, { account: { userId: user._id } });
-		expect(await cb).to.be.true;
+		expect(nestMailService.send.calledOnce).to.eql(true);
```

Comment:
is the eql operator the right one for an bool value?

Reply:
all occurences have been replaced by equal (resolved them)